src/pci.c

```diff
@@ -163,21 +177,22 @@ void pci_dev_register(struct pci_dev *dev)
 {
     /* FIXEME: It just simplifies the registration on pci bus 0 */
     /* FIXEME: dev_num might exceed 32 */
-    union pci_config_address addr = {.enable_bit = 1,
-                                     .dev_num = dev->pci_bus->dev_num};
+    union pci_config_address addr = {.dev_num = dev->pci_bus->dev_num};
     dev_init(&dev->config_dev, addr.value, PCI_CFG_SPACE_SIZE, dev,
              pci_config_do_io);
     bus_register_dev(dev->pci_bus, &dev->config_dev);
 }
 
 #define PCI_CONFIG_ADDR 0xCF8
 #define PCI_CONFIG_DATA 0xCFC
+#define PCI_CONFIG_SIZE (1UL << 16)
```

Comment:
It is confusing to name it using the `PCI_CONFIG_*` convention. I'll recommend to rename to something different like `PCI_MMIO_SIZE`